frontend/webpack.config.js

```diff
@@ -56,6 +63,14 @@ module.exports = {
 	},
 	plugins: [
 		new HtmlWebpackPlugin({ template: "src/index.html.ejs" }),
+		new webpack.ProvidePlugin({
+			process: "process",
+		}),
+		new webpack.DefinePlugin({
+			"process.env.REACT_APP_BASE_URL": JSON.stringify(
+				process.env.REACT_APP_BASE_URL,
+			),
+		}),
```

Comment:
How about just passing all env keys:

```suggestion
		new webpack.DefinePlugin({
			"process.env": JSON.stringify(dotenv.config().parsed),
		}),
```